config.json

```diff
@@ -1,41 +1,42 @@
 {
   "pagesRepository": "tldr-pages/tldr",
   "url": "http://tldr-pages.github.io/assets/tldr.zip",
-  "colors": {
-    "text": "green",
-    "command-background": "black",
-    "command-foreground": "red",
-    "command-token": "white"
-  },
   "themes": {
-    "bw": {
-      "name": "bold, underline",
+    "simple": {
+      "name": "bold,underline",
```

Comment:
If it makes no difference, then I'd suggest keeping the spaces, since it's more readable.